services/app-graphql/src/schema.graphql

```diff
@@ -26,6 +26,21 @@ type Query {
         input: FetchHealthPlanPackageInput!
     ): FetchHealthPlanPackagePayload!
 
+    """
+    fetchHealthPlanPackage returns a specific HealthPlanPackage with Q&A questions by id
+
+    If a package with the given ID cannot be found, this query returns undefined
+    CMS users cannot fetch a DRAFT HealthPlanPackage
+
+    Errors:
+    - ForbiddenError:
+    - A state user from a different state called this.
+    - A CMSUser attempted to fetch a DRAFT HealthPlanPackage
+    """
+    fetchHealthPlanPackageWithQuestions(
```

Comment:
This should not be neccessary. we should still have a single fetchHealthPlanPackage Query that can be called either requesting questions or not.